tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import pytest
+
+import lkml
+import lkml.tokens as tokens
+
+
+@pytest.fixture
```

Comment:
I would prefer these fixtures stay in their respective test files instead of in another layer of configuration. These fixtures aren't being used beyond their individual files so this isn't worth the extra cognitive overhead of having them in another file.

Reply:
👍